src/types/fetch.ts

```diff
@@ -3,6 +3,8 @@ import type { FetchRequest, FetchOptions, FetchResponse } from 'ohmyfetch'
 // An interface to extend in a local project
 export interface InternalApi { }
 
+export type NitroFetchRequest = keyof InternalApi | (`${string}${'/'|'.'}${string}` & {}) | Exclude<FetchRequest, string>
```

Comment:
I think we can drop `.`:

```suggestion
export type NitroFetchRequest = keyof InternalApi | (`${string}${'/'}${string}` & {}) | Exclude<FetchRequest, string>
```